enterprise/management/commands/migrate_enterprise_catalogs.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+Django management command for migrating EnterpriseCustomerCatalog data to new service.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import logging
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.translation import ugettext as _
+
+from enterprise.api_client.enterprise_catalog import EnterpriseCatalogApiClient
+from enterprise.models import EnterpriseCustomerCatalog
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Migrate EnterpriseCustomerCatalog data to new Enterprise Catalog service.
+    """
+    help = 'Migrate EnterpriseCustomerCatalog data to new Enterprise Catalog service.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--api_user',
+            dest='api_user',
+            required=True,
+            metavar='LMS_API_USERNAME',
+            help=_('Username of a user authorized to access the Enterprise Catalog API.'),
+        )
+        super(Command, self).add_arguments(parser)
+
+    def handle(self, *args, **options):
+        api_username = options['api_user']
+        try:
+            user = User.objects.get(username=api_username)
+        except User.DoesNotExist:
+            raise CommandError(_('A user with the username {username} was not found.').format(username=api_username))
+
+        client = EnterpriseCatalogApiClient(user=user)
+        for enterprise_catalog in EnterpriseCustomerCatalog.objects.all():
+            LOGGER.info('Migrating Enterprise Catalog {}'.format(enterprise_catalog.uuid))
```

Comment:
nitpick about wording, but i would say "create" here too instead of "migrate" (which feels like this record would somehow be gone after the task is complete)